### ISSUE TITLE
Update go version

### ISSUE THREAD

**Issue by wvxiw8** [CSR-LC/csr-be#80]
Update go version to the latest one (1.22 at the moment)

**Comment by wvxiw8** [comment]
As @smailic05 reported the coverage decreased after updating the version and that's why we cannot merge the code unless more tests added. 
@WolfusFlow maybe we can lower the coverage level and merge the code? It's an artificial metric and having coverage decreased just a psychological moment in our case. @smailic05 is busy on the prod project and we just postpone the merge.

**Comment by wvxiw8** [comment]
@smailic05 if you added new tests, but hadn't pushed them, you can do it. Did you?
